tests/Serializer/AbstractSerializerTest.php

```diff
@@ -65,6 +65,34 @@ public function testObjectsAreNotStrings(): void
         $this->assertSame(['key' => 'value'], $result);
     }
 
+    public function traversableDataProvider(): \Generator
```

Comment:
Please move this method below the test method, and replace the `traversable` word with `iterable`